app/decorators/user_decorator.rb

```diff
@@ -40,7 +40,7 @@ def masked_two_factor_phone_number
   end
 
   def identity_not_verified?
-    user.active_profile ? false : true
+    !user.verified?
```

Comment:
I think I prefer the later, but maybe i'm not ruby enough.   Whatever you decide is fine with me.